### PR TITLE
add openvr binding for manual flashlight

### PR DESCRIPTION
**Pull request by drowhunter**
Allow Manual Flashlight to be bound to a controller using OpenVR input

### CODE REVIEW

**Comment by drowhunter** [comment]
Works really well when I set toggle_flashlight on hold of the X button (Left A on index)

File: src/mods/VR.cpp [hunk 1]
```diff
@@ -2697,6 +2714,9 @@ void VR::openvr_input_to_re2_re3(REManagedObject* input_system) {
     // Change Ammo
     set_button_state(app::ropeway::InputDefine::Kind::CHANGE_BULLET, is_change_ammo_down);
 
+    // Toggle Flashlight
+    set_button_state(app::ropeway::InputDefine::Kind::FLASHLIGHT, is_toggle_flashlight_down);
```
Comment (praydog) [inline src/mods/VR.cpp:2718]: You cannot add non-existent bindings to the game's enums. Please remove these lines.

File: src/sdk/Enums_Internal.hpp [hunk 2]
```diff
@@ -31591,6 +31591,7 @@ namespace app::ropeway::InputDefine {
         UI_MAP_DOWN = 140737488355328,
         UI_SORT = 281474976710656,
         UI_DISP = 562949953421312,
+        FLASHLIGHT = 1125899906842624
```
Comment (praydog) [inline src/sdk/Enums_Internal.hpp:31594]: You cannot add non-existent bindings to the game's enums. Please remove this line.

**Review by praydog — CHANGES_REQUESTED** [review]

File: src/mods/VR.cpp (hunk 1 above)
Comment (drowhunter) [reply src/mods/VR.cpp:2718]: ok no problem,  I didn't know what this line actually does.

File: src/sdk/Enums_Internal.hpp (hunk 2 above)
Comment (drowhunter) [reply src/sdk/Enums_Internal.hpp]: ok removed offending line

**Comment by praydog** [comment]
Merged. Thanks for the PR!

File: src/mods/VR.cpp (hunk 1 above)
Comment (praydog) [reply src/mods/VR.cpp]: Yes, the app::ropeway::InputDefine::Kind enum belongs to the game. `set_button_state` is only for already existing in-game actions.

**Pull request merged**